resources/assets/uad_lists.json

```
@@ -16236,11 +16236,11 @@
   },
   "com.xiaomi.bluetooth.overlay": {
     "list": "Oem",
-    "description": "MIUI Bluetooth Control.\nYou need to keep this if you want the bluetooth to work.",
+    "description": "It has only unused png files, webp.\nImages in it: wireless headphones from all angles.",
     "dependencies": [],
     "neededBy": [],
     "labels": [],
-    "removal": "Expert"
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```
If the earbud's image doesn't show up anymore for users, it may confuse them and it's hard to get back as users usually uninstall the packages.

---

i never seen these images maybe unused or for specific xiaomi earbuds, but its only app with png, webp files

---

so advanced i guess, no trackers, no activities